src/containers/SubjectPage/SubjectPage.jsx

```diff
@@ -31,13 +31,25 @@ const SubjectPage = ({
     ndlaFilm,
     match,
   });
+
+  const [currentTopic, setCurrentTopic] = useState(undefined);
+
   const { loading, data } = useGraphQuery(subjectPageQueryWithTopics, {
     variables: {
       subjectId,
-      topicId: topicId || '',
+      topicId: !currentTopic && topicId ? topicId : '',
```

Comment:
Synes fremdeles det er uheldig å sende inn tom streng her. Det gir en feilmelding tilbake fra graphql: "Cannot return null for non-nullable field Topic.id."
Det hadde vore bedre å legge på en boolsk verdi her og enten bruke `@skip(if: $skipTopic)` eller ` @include(if: $includeTopic)`.